public/docs/getting_started/compile.md

```diff
@@ -1,43 +1,59 @@
 
-To compile your contracts, simply run:
+With a bare Truffle [project](./project) (created through `truffle init`), you're given a single `Migrations.sol` file that is responsible for deploying your contracts to the blockchain. If you're using a [Truffle Box](/boxes), you will have multiple files here.
 
-```none
-$ truffle compile
+## Command
+
+If you are using Truffle Develop, type the following into the Truffle Develop console:
+
+```shell
+compile
+```
+
+Otherwise, in a terminal, type the following:
+
+```shell
+truffle compile
 ```
 
-Truffle will compile only the contracts that have been changed since the last compile, to reduce any unnecessary compilation. If you'd like to override this behavior, run the above command with the `--all` option.
+Upon first run, all contracts will be compiled. Upon subsequent runs, Truffle will compile only the contracts that have been changed since the last compile. If you'd like to override this behavior, run the above command with the `--all` option.
+
+## Build artifacts
 
-# Artifacts
+Artifacts of your compilation will be placed in the `build/contracts/` directory, relative to your project root. (This directory will be created if it does not exist.)
 
-Artifacts of your compilation will be placed in the `./build/contracts` directory, relative to your project. This directory will be created if it does not exist. These artifacts are integral to the inner workings of Truffle, and they play an important part in the successful deployment of your application. You should not edit these files by hand as they'll be overwritten by contract compilation and deployment.
+These artifacts are integral to the inner workings of Truffle, and they play an important part in the successful deployment of your application. **You should not edit these files** as they'll be overwritten by contract compilation and deployment.
 
-# Dependencies
+## Dependencies
 
 You can declare contract dependencies using Solidity's [import](http://solidity.readthedocs.org/en/latest/layout-of-source-files.html#importing-other-source-files) command. Truffle will compile contracts in the correct order and ensure all dependencies are sent to the compiler. Dependencies can be specified in two ways:
 
-### Importing dependencies via their file name:
+### Importing dependencies via file name
 
-To import contracts from a separate file, simply write the following command, where `AnotherContract.sol` is relative to the current contract being written. This will make all contracts within `AnotherContract.sol` available to the current source file.
+To import contracts from a separate file, simply use the following command:
 
 ```
 import "./AnotherContract.sol";
 ```
 
+This will make all contracts within `AnotherContract.sol` available to the current source file. Here, `AnotherContract.sol` is relative to the path of the current contract being written.
```

Comment:
Aside: If you take my suggestion above word for word, we're using `source file` twice. I'd change one or the other in that case.